html/index.html

```diff
@@ -0,0 +1,480 @@
+        
+                        <img src="../assets/h5Logo.png" alt="">
+        
+                        <div id="html">
+                            
+                            <span> HTML</span>
+        
+                        </div>
+        
+                    </div>
+        
+                    <div class="language-css">
+        
+                        <img src="../assets/css3Logo.png" alt="">
+        
+                        <div id="css"> 
+                            
+                            <span> CSS </span>
+        
+                        </div>
+        
+                    </div>
+        
+                </div>
+        
+                <div class="language-footer">
+        
+                    <div class="language-framwork">
+        
+                        <p class="bold-text">Frameworks</p>
+        
+                        <img src="../assets/dropdown.png" alt="" class="dropside-img">
+            
+                    </div>
+        
+                    <hr id="hr">
+        
+                    <div class="language-skill">
+        
+                        <p  class="bold-text">Skills</p>
+        
+                        <img src="../assets/dropdown.png" alt="" class=" dropside-img">
+            
+                    </div>
+        
+                    <hr>
+        
+        
+        
+                </div>
+        
+        
+            </div>
+
+            <div class="section">
+    
+                <div class="contact-me-section">
+        
+                    <p id="contact-me">Contact me</p>
+        
+                    <p id="contact-msg"> 
+                        If you have an application you are 
+                        interested in developing, a feature 
+                        that you need built or a project that
+                        needs coding. I’d love to help with it 
+                    </p>
+        
+                </div>
+        
+        
+                <div class="form-section">
+        
+                    <form action="">
+        
+                        <input type="text" placeholder="Toyyib" class="text"> <br>
+                        <input type="email" placeholder="Simpleshaikh2@gmail.com" class="text"> <br>
+                        <textarea type="text" placeholder="write your message heres" id="type-msg"> </textarea><br>
+        
+                    </form>
+        
+                    <div class="btn-sec">
+        
+                        <button class="sec-btn">   
+                            Get in touch
+                        </button>
+            
+                        <span class="dot"></span>
+        
+                    </div>
+                    
+        
+                </div>
+        
+                <div class="black-linen">
+                    <img src="../assets/Home Indicator/Shape.png" alt="">
+                </div>
+        
+            </div>
+    
+        </div>
```

Comment:
- [ ] Kindly remove this section as it's not part of the requirement for this milestone. You can keep it on some other local file and add it to your next PR✔
- Please restrain from adding contents that are part of future milestones as this will cause much confusion when checking the "files changed" section.